web3/eth/eth.py

```diff
@@ -281,6 +281,22 @@ def _durin_call(
 
         raise TooManyRequests("Too many CCIP read redirects")
 
+    # eth_createAccessList
+
+    _create_access_list: Method[
+        Callable[
+            [TxParams, Optional[BlockIdentifier]],
+            TxReceipt,
```

Comment:
We should use an appropriate return type here as per [this comment](https://github.com/ethereum/web3.py/pull/2381/files#r1416119226).